web-client/src/ducks/requests/actions.ts

```diff
@@ -3,37 +3,49 @@ import { IRequest, Request } from 'src/models/requests';
 /* TODO: why rename? */
 import {
   createUserRequest,
-  getAcceptedPost as getAcceptedPostFunc,
-  getOpenPost as getOpenPostFunc,
+  getCavRequestPosts as getCavPostFunc,
+  getFindPosts as getFindPostsFunc,
+  getPinReqestPosts as getPinRequestPostsFunc,
   setUserRequest,
 } from './functions';
 import {
   CHANGE_MODAL,
-  GET_OFFER_POST,
-  GET_REQUEST_POST,
+  GET_CAV_REQUEST_POSTS,
+  GET_FIND_REQUEST_POSTS as GET_FIND_POSTS,
+  GET_PIN_REQUEST_POSTS,
   IgetRequestPosts,
-  RESET_OFFER_POST,
-  RESET_REQUEST_POST,
+  RESET_CAV_REQUEST_POSTS,
+  RESET_FIND_REQUEST_POSTS,
+  RESET_PIN_REQUEST_POSTS,
   RESET_SET,
   SET,
   SET_TEMP_REQUEST,
 } from './types';
 
-export const getRequestPosts = (payload: IgetRequestPosts) => (
+export const getFindPosts = (payload: IgetRequestPosts) => (
   dispatch: Function,
 ) =>
   dispatch({
-    type: GET_REQUEST_POST,
-    firebase: getOpenPostFunc,
+    type: GET_FIND_POSTS,
+    firebase: getFindPostsFunc,
     payload,
   });
 
-export const getOfferPosts = (payload: IgetRequestPosts) => (
+export const getPinRequestPosts = (payload: IgetRequestPosts) => (
   dispatch: Function,
 ) =>
   dispatch({
-    type: GET_OFFER_POST,
-    firebase: getAcceptedPostFunc,
+    type: GET_PIN_REQUEST_POSTS,
+    firebase: getPinRequestPostsFunc,
+    payload,
+  });
+
+export const getCavRequestPosts = (payload: IgetRequestPosts) => (
```

Comment:
These are the request posts for which the user responded by making offers right? just confirming. 
Can you add this as a comment on top of the name of the function?

Reply:
I have put it into code for my next pull request.